src/audio/selector/selector.c

```diff
@@ -633,6 +654,7 @@ static void set_selector_params(struct processing_module *mod,
 	struct comp_dev *dev = mod->dev;
 	struct comp_data *cd = module_get_private_data(mod);
 	struct comp_buffer __sparse_cache *source;
+	struct sof_selector_ipc4_config sel_cfg;
 	struct ipc4_audio_format *out_fmt;
```

Comment:
same here: make these to `const` pointers
```
	const struct sof_selector_ipc4_config *sel_cfg = &cd->sel_ipc4_cfg;
	const struct ipc4_audio_format *out_fmt = &sel_cfg->output_format.audio_fmt;
```

Reply:
All reviews are addressed, thanks.